packages/openapi-fetch/package.json

```diff
@@ -9,6 +9,16 @@
   "license": "MIT",
   "type": "module",
   "main": "./dist/index.js",
+  "module": "./dist/index.js",
```

Comment:
Also I’m fine with adding this, but this is more of an “invalid” / “legacy” property which was never officially standardized, which was why it was missing in the first place. It’s also not needed because it’s redundant with `"main"`, but it’s also fine to include just in case it helps something.